docs/infrastructure/hosted-server.rst

```diff
@@ -84,12 +84,31 @@ When creating a new VM from a template…
 #. Start VM, follow any setup scripts
 #. Profit!
 
+
+**************
 Use and Access
-==============
+**************
 
-Use and access is limited to members of the executive board.
+Use and access of Titan is limited to the President, Vice President, and Project Coordinator.
 In the future, there may be opportunities for club members to participate in managing RITlug's infrastructure.
 
+How to Gain Access to Titan
+===========================
+
+If you are one of the three roles listed above, and have not yet gained access to Titan, this is the section for you.
+Access to Titan first requires access to the VPN, as Titan's Management Interface (TMI) is IP restricted.
+To get on the VPN, Solomon needs to give you the OpenVPN file to initiate the connection.
+This VPN does not allow anything but local traffic through, so tunneling does not work.
+
+Lastly, TMI needs a (TMI) proxmox account. 
+Proxmox is used in order to avoid unnecessary unix accounts.
+This requires a username, and Solomon provides a temporary password that should be changed upon log in.
+The steps for this process are listed below:
+
+#. Start a conversation with Solomon Rubin
```

Comment:
Since not everyone reads the entire page, I'd like to add reiterate that this is only for the specified people again. I don't want Solomon getting PMed constantly by people who didn't read the page. Moving L98 here may be a good fix

Reply:
We can add a clause "For approved users this is the process"

Reply:
@ct-martin do you really think it needs to be said a third time? I mention it in the first paragraph, and also in the first sentence of the "Use and Access" section.

Reply:
@Tjzabel no, which is why I suggested moving that line instead of copying it again. So L98 would get removed from it's current place and get inserted between L107 & L108 instead (and then tweak slightly to keep a good flow)

Reply:
Done